README.md

```diff
@@ -81,7 +81,7 @@ This will make use of `Microsoft.AspNetCore.SystemWebAdapters` to provide the `S
     app.UseSession();
     app.UseSystemWebAdapters();
     ...
-    app.MapWebForms(); // Required for pages
+    app.MapHttpHandlers(); // Required for pages (or other configured handlers)
```

Comment:
let's document.. in next release.. as we might have users using this way

Reply:
The README is updated, but I'll mark this as a breaking change